srcpkgs/docker-cli/template

```diff
@@ -0,0 +1,39 @@
+# Template file for 'docker-cli'
+pkgname=docker-cli
+version=20.10.5
+revision=1
+archs="aarch64* armv[67]l* x86_64* ppc64le*"
+create_wrksrc=yes
+build_style=go
+go_import_path="github.com/docker/cli"
+hostmakedepends="pkg-config tar"
+depends="containerd runc iptables xz git"
+short_desc="Pack, ship and run any application as a lightweight container - cli"
+maintainer="PWA COLLECTIVE <mail@pwa.cya.cx>"
+license="Apache-2.0"
+homepage="http://www.docker.io"
+distfiles="https://github.com/docker/cli/archive/v${version}.tar.gz"
+checksum=4ba845f8c7e2e0a2ca1ec6589847159ca8d0d37b609f0e6f78def7a893b9b342
+system_groups="docker"
+
+_cli_commit=55c4c88966a912ddb365e2d73a4969e700fc458f
+_cli_builddir="cli-$version"
+
+export GO111MODULE=off # go1.16 defaults to on
+
+do_build() {
+	export AUTO_GOPATH=1
+	export GITCOMMIT=$_cli_commit
+	export DISABLE_WARN_OUTSIDE_CONTAINER=1
+
+	mkdir -p "$wrksrc/_cli_builddir"
+
+	cd "$wrksrc/$_cli_builddir"
+	mkdir -p src/github.com/docker/
+	ln -sf "$wrksrc/$_cli_builddir" src/github.com/docker/cli
+	GOPATH="$PWD" LDFLAGS="" VERSION="$version" make dynbinary
+}
+
+do_install() {
+	vbin $_cli_builddir/build/docker
+}
```

Comment:
Especially if you can drop the custom `do_build`, I wonder if you can drop this custom function too.

Reply:
As mentioned in the other thread, I attempted this and failed, so for now I'll leave this as-is.

Reply:
Resolved